OpenRA.Game/Sound/Sound.cs

```diff
@@ -48,30 +47,12 @@ static ISoundSource LoadSound(string filename)
 
 		static ISoundSource LoadWave(WavLoader wave)
 		{
-			return soundEngine.AddSoundSourceFromMemory(wave.RawOutput, wave.Channels, wave.BitsPerSample, wave.SampleRate);
+			return Engine.AddSoundSourceFromMemory(wave.RawOutput, wave.Channels, wave.BitsPerSample, wave.SampleRate);
 		}
 
 		static ISoundSource LoadSoundRaw(byte[] rawData, int channels, int sampleBits, int sampleRate)
 		{
-			return soundEngine.AddSoundSourceFromMemory(rawData, channels, sampleBits, sampleRate);
-		}
-
-		static ISoundEngine CreateEngine(string engine)
```

Comment:
Ah, i'm basically just asking you to keep this, but fetch from the platform assembly instead.